File: fix2.py
import re

with open('app/scenes/page.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

# 修复列表视图的 SceneSortable - 改为自闭合标签
old_text = '''            <SceneSortable<Scene>
              items={filteredScenes}
              onChange={handleReorderScenes}
              className="space-y-3"
              renderItem={(scene: Scene) => (
                <div key={scene.id} className="group pl-8">
                  <SceneCard
                    scene={scene}
                    onEdit={handleEditScene}
                    onDelete={handleDeleteScene}
                  />
                </div>
              )}
            </SceneSortable>'''

new_text = '''            <SceneSortable<Scene>
              items={filteredScenes}
              onChange={handleReorderScenes}
              className="space-y-3"
              renderItem={(scene: Scene) => (
                <div key={scene.id} className="group pl-8">
                  <SceneCard
                    scene={scene}
                    onEdit={handleEditScene}
                    onDelete={handleDeleteScene}
                  />
                </div>
              )}
            />'''

if old_text in content:
    content = content.replace(old_text, new_text)
    with open('app/scenes/page.tsx', 'w', encoding='utf-8') as f:
        f.write(content)
    print('Fixed list view SceneSortable!')
else:
    print('Pattern not found for list view')

File: fix2.js
const fs = require('fs');
let content = fs.readFileSync('app/scenes/page.tsx', 'utf-8');

// 修复看板视图 - className 属性后面缺少 >
content = content.replace(
  /className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-4"\s+renderItem/,
  'className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-4"\n                  renderItem'
);

// 更精确的修复：在 className 闭合引号后添加 >
content = content.replace(
  /(className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-4")\s+(renderItem)/,
  '$1\n                  $2'
);

fs.writeFileSync('app/scenes/page.tsx', content);
console.log('Fixed!');
